### PR TITLE
Media Links Docs Updates

### PR DESCRIPTION
**Pull request by MatkoTom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change.

Updated the media, links and docs UI. Added some new styles.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration. All testing was done
manually by following, plainly, functionality requirements. If there are some special edge cases or
special ways that things need to be tested they will be mentioned below.

Dev testing

**Test Configuration**:

* Firmware version: A556BXXS3AXF2
* Hardware: SAMSUNG Galaxy A55
* SDK: Android 14

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

Media(6)
Media(7)

### CODE REVIEW

**Review by ivolarevic — CHANGES_REQUESTED** [review]
Please, double check text sizes  in Media, Links and Docs screen.

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/DocsFragment.kt [hunk 1]
```diff
@@ -57,8 +58,11 @@ class DocsFragment(private val roomsWithUsers: RoomWithUsers?) : BaseFragment()
                     when (state) {
                         is MediaScreenState.Success -> {
                             if (state.media != null) {
```
Comment (ivolarevic) [inline app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/DocsFragment.kt:60]: isNotEmpty() == true

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/LinksFragment.kt [hunk 2]
```diff
@@ -57,8 +57,11 @@ class LinksFragment(private val roomsWithUsers: RoomWithUsers?) : BaseFragment()
                     when (state) {
                         is MediaScreenState.Success -> {
                             if (state.media != null) {
```
Comment (ivolarevic) [inline app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/LinksFragment.kt:59]: same as above

File: app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/MediaDetailsFragment.kt [hunk 3]
```diff
@@ -119,8 +119,11 @@ class MediaDetailsFragment(private val roomsWithUsers: RoomWithUsers?) : BaseFra
                     when (state) {
                         is MediaScreenState.Success -> {
                             if (state.media != null) {
+                                binding.tvNoMedia.visibility = View.GONE
```
Comment (ivolarevic) [inline app/src/main/java/com/clover/studio/spikamessenger/ui/main/chat/media_links_docs/MediaDetailsFragment.kt:122]: same as above

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -65,5 +66,6 @@
     <dimen name="reply_layout_dimen">128dp</dimen>
     <dimen name="one_hundred_fifty_six">156dp</dimen>
     <dimen name="ten_sp_text">10sp</dimen>
+    <dimen name="ninety_six_dp">96dp</dimen>
```
Comment (ivolarevic) [inline app/src/main/res/values/dimens.xml:69]: can you set this value under   tag?

**Review by ivolarevic — APPROVED** [review]
LGTM
